apps/files/src/store/state.js

```diff
@@ -19,7 +19,8 @@ export default {
     '{http://owncloud.org/ns}size',
     '{DAV:}getlastmodified',
     '{DAV:}getetag',
-    '{DAV:}resourcetype'
+    '{DAV:}resourcetype',
+    '{http://owncloud.org/ns}downloadURL'
```

Comment:
Is the download URL available only for public links or also for private files?

Reply:
Hehehehehe, I asked the same thing in the chat. It is not. I would love to have all download urls already by the server. Then we could get rid of the url signing in the sdk and would not need to make any difference between public and private context.

Reply:
Hehe. Exactly my thoughts!